src/utils/helpers.js

```diff
@@ -109,24 +109,6 @@ const helpers = {
     }
   },
 
-  /**
-   * Render value to integer or decimal
-   *
-   * @param {number} amount Amount to render
-   * @param {boolean} isInteger If it's an integer or decimal
-   *
-   * @return {string} rendered value
-   * @memberof Helpers
-   * @inner
-   */
-  renderValue(amount, isInteger) {
-    if (isInteger) {
-      return hathorLib.numberUtils.prettyIntegerValue(amount);
-    } else {
-      return hathorLib.numberUtils.prettyValue(amount);
-    }
-  },
-
```

Comment:
This was changed to use only `prettyValue` since it now can handle integer (decimals = 0)